content/articles/oop-in-java-abstraction-and-encapsulation/index.md

```diff
@@ -1,350 +1,482 @@
-#**The concept of OOP In Java Getting Started with ABSTRACTION and ENCAPSULATION**
+# The concept of OOP In Java Getting Started with ABSTRACTION and ENCAPSULATION
 
-In Java, everything is considered an object and is treated as such; a real-life entity.
+In Java, everything is an object and as such; a real-life entity.
 
-Object-Oriented Programming(OOP) unlike Procedural programming is an approach or a programming paradigm where everything is treated as Classes(Logical construct) and Objects(Real-life entities or instances of a class).
+Object-Oriented Programming (OOP) unlike Procedural programming is an approach. Or a programming paradigm where everything is an Object (Real-life entities). OOP tries to model real-life objects.
+Objects come from class, a form of generalization but the real thing is the Object. What then is a class?
 
-Class is a template from which objects are created. 
-The process of creating Objects from a class is referred to as Instantiation
+Class is a template from which objects are created 
 
-In this tutorial, you will learn about Abstraction and Encapsulation and build more on your already existing knowledge of the concept of OOP, and to say the least, this will be a nice piece to start with for absolute java beginners.
+The process of creating Objects from a class is Instantiation
+
+In this tutorial, you will learn about Abstraction, Encapsulation, Objects, and classes. 
```

Comment:
don't capitalize the words *objects*, *abstraction* and *encapsulation* they are not [proper nouns](https://www.collinsdictionary.com/dictionary/english/proper-noun).